src/test/ui/mir-dataflow/indirect-mutation-1.rs

```diff
@@ -0,0 +1,33 @@
+#![feature(core_intrinsics, rustc_attrs, const_raw_ptr_deref)]
+
+use std::cell::Cell;
+use std::intrinsics::rustc_peek;
+
+#[rustc_mir(rustc_peek_indirectly_mutable, stop_after_dataflow)]
+pub fn mut_ref(flag: bool) -> i32 {
+    let mut i = 0;
+    let cell = Cell::new(0);
+
+    if flag {
+        let p = &mut i;
+        unsafe { rustc_peek(i) };
+        *p = 1;
+
+        let p = &cell;
+        unsafe { rustc_peek(cell) };
+        p.set(2);
+    } else {
+        unsafe { rustc_peek(i) };    //~ ERROR rustc_peek: bit not set
+        unsafe { rustc_peek(cell) }; //~ ERROR rustc_peek: bit not set
+
+        let p = &mut cell;
```

Comment:
`cell` is immutable, how does this line not error?

Reply:
`stop_after_dataflow` prevents later all subsequent compiler passes from running, so we never get to the part where this would error. I'll make sure these tests run outside of `rustc_peek` mode.